README.md

```diff
@@ -145,7 +145,7 @@ more Emacs-y.
 
         apt install texinfo libjpeg-dev libtiff-dev \
           libgif-dev libxpm-dev libgtk-3-dev libgnutls28-dev \
-          libncurses5-dev libxml2-dev libxt-dev
+          libncurses5-dev libxml2-dev libxt-dev clang-7
```

Comment:
Instead of putting it here, let's update the section right above it where it suggests installing `build-essential`. It also suggests installed `clang` which in my case at least should have brought in `clang-7`. Let's break out that apt section to resemble the section below so it is clear that they should run this. Also, let's add `clang-dev` since people usually miss it.

Reply:
By `clang-dev` I assume you're referring to `libclang-dev`?